node/src/components/consensus/highway_core.rs

```diff
@@ -35,6 +35,7 @@ mod test_macros;
 pub(crate) mod active_validator;
 pub(crate) mod finality_detector;
 pub(crate) mod highway;
+mod lnc;
```

Comment:
There's a block with private modules below.

Reply:
Done in [`4052f72` (#572)](https://github.com/CasperLabs/casper-node/pull/572/commits/4052f72d1f3a987729feb79886716d40dc756ff6).